openfl/interface/collaborator.py

```diff
@@ -25,7 +25,7 @@
 from openfl.interface.cli_helper import CERT_DIR
 from openfl.utilities.path_check import is_directory_traversal
 from openfl.utilities.utils import rmtree
-
+from openfl.utilities.attestation import attestation_utils as attestation_utils
```

Comment:
same here?

Reply:
there are some generic util methods which are part of attestation_utils and not AttestationManager like fetch_attestation_env_vars. Hence importing such